app/models/manageiq/providers/ansible_tower/shared/automation_manager/configuration_script_source.rb

```diff
@@ -8,36 +8,68 @@ def create_in_provider(manager_id, params)
         connection.api.projects.create!(params)
       end
 
+      refresh(manager)
+      find_by!(:manager_id => manager.id, :manager_ref => project.id)
+    end
+
+    def create_in_provider_queue(manager_id, params)
+      manager = ExtManagementSystem.find(manager_id)
+      queue(manager.my_zone, nil, "create_in_provider", [manager_id, params], "Creating #{name}")
```

Comment:
@blomquisg `name` or `manager_ref`(the native Tower ID), or `id` (vmdb record ID), which one is better?